monitor_check.py

```diff
@@ -279,6 +335,52 @@ def bug(bugs, package):
 
 counter = Counter()
 
+def pkgname(srcname):
```

Comment:
This is **really** confusing.

`srcname` is not a name of src package (e.g. `python-setuptools`), but a NEVRA of the built package (e.g. `python3-setuptools-53.0.0-1.fc34.noarch`).

`pkgname` returns the name of the source package, not the name of the package.

I'd call the function `source_name` and the argument `nevra`.